cmd/opm/serve/serve.go

```diff
@@ -103,9 +123,121 @@ func (s *serve) run(ctx context.Context) error {
 	health.RegisterHealthServer(grpcServer, server.NewHealthServer())
 	reflection.Register(grpcServer)
 	s.logger.Info("serving registry")
+	p.stopCpuProfileCache()
```

Comment:
So the goal is to only log the cpu profile for the startup?

Reply:
Correct.  The pprof endpoint does not cache any information until/unless connected-to and requested to do so.  This generates a cache of the CPU profile data from init until the grpc server is ready, accessible for the lifetime of the session.